docs/quickstart.md

```diff
@@ -50,32 +48,32 @@ PostgreSQL database at localhost:5432.
 ```shell
 #!/usr/bin/env sh
 
-VERSION=4
+VERSION=6
 
-#Get the Gateway Jar
-curl https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/${VERSION}/gateway-ha-${VERSION}-jar-with-dependencies.jar -o ./gateway-ha.jar
-
-#Start the persistence DB
-if ! $(command -v psql >/dev/null); then
-    if ! $(command -v brew >/dev/null); then
-        echo 'Please install psql and rerun';
-        exit 1
-    else
-        brew install libpq
-    fi
+#Check and get the Gateway Jar
+if [[ -f "gateway-ha.jar" ]]; then
+    echo "Found gateway-har.jar file in current directory."
+else
+    echo "Failed to find gateway-ha.jar in current directory. Fetching version $VERSION from Maven Central repository."
+    curl https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/${VERSION}/gateway-ha-${VERSION}-jar-with-dependencies.jar -o ./gateway-ha.jar
 fi
 
-export PGPASSWORD=mysecretpassword
-docker run --name local-postgres -p 5432:5432 -e POSTGRES_PASSWORD=$PGPASSWORD -d postgres:latest
-#make sure the DB has time to initialize
-sleep 5
-
-#Initialize the DB
-curl https://raw.githubusercontent.com/trinodb/trino-gateway/main/gateway-ha/src/main/resources/gateway-ha-persistence-postgres.sql > ./gateway-ha-persistence-postgres.sql 
-psql -U postgres -h localhost -c 'CREATE DATABASE gateway'
-psql  -U postgres -h localhost -d gateway -f ./gateway-ha-persistence-postgres.sql 
+#Check if DB is running
+if docker ps --format '{{.Names}}' | grep -q '^local-postgres$'; then
+    echo "PostgreSQL database container 'localhost-postgres' is already running. Only starting Trino Gateway."
+else
+    echo "PostgreSQL database container 'localhost-postgres' is not running. Proceeding to initialize and run database server."
+    export PGPASSWORD=mysecretpassword
+    docker run -v ./gateway-ha/src/main/resources/gateway-ha-persistence-postgres.sql:/tmp/gateway-ha-persistence-postgres.sql --name local-postgres -p 5432:5432 -e POSTGRES_PASSWORD=$PGPASSWORD -d postgres:latest
```

Comment:
this assumes the project checkout root folder `trino-gateway` is the location of the quickstart script